src/core/gui/PdfFloatingToolbox.cpp

```diff
@@ -81,7 +85,6 @@ auto PdfFloatingToolbox::getOverlayPosition(GtkOverlay* overlay, GtkWidget* widg
         allocation->width = natural.width;
         allocation->height = natural.height;
 
-        if (self->pdfElemSelection && self->pdfElemSelection->getPageView()) {
             // Make sure the "pdfFloatingToolbox" is fully displayed.
```

Comment:
The indentation is messed up here.

Reply:
Uh... Weird that git-clang-format is not picking that out. I fixed it.